ci/gpu/build.sh

```diff
@@ -180,31 +165,17 @@ else
     gpuci_logger "Check GPU usage"
     nvidia-smi
 
-    gpuci_logger "Update binaries"
-    cd $LIBCUML_BUILD_DIR
-    chrpath -d libcuml.so
-    chrpath -d libcuml++.so
-    patchelf --replace-needed `patchelf --print-needed libcuml++.so | grep faiss` libfaiss.so libcuml++.so
+    gpuci_mamba_retry install -y -c ${CONDA_ARTIFACT_PATH} libcuml-tests
```

Comment:
Based on the comment below, it might help improve transparency if we also include `libcuml` in this install command.

- https://github.com/rapidsai/cudf/pull/10326#discussion_r829346769

Reply:
Done